### PR TITLE
Updated ENSOInfluence notebook

### PR DESCRIPTION
**Pull request by MatthewJA**
Two updates to the ENSOInfluence notebook:

1. Updated to use `dea_waterbodies.get_time_series` instead of loading and parsing waterbody time series itself
2. Added seasonal checks and deseasonalising

### Checklist (replace `[ ]` with `[x]` to check off)
- [x] Notebook created using the [DEA-notebooks template](https://github.com/GeoscienceAustralia/dea-notebooks/tree/develop)
- [x] Remove any unused Python packages from `Load packages`
- [x] Remove any unused/empty code cells
- [x] Remove any guidance cells (e.g. `General advice`)
- [x] Ensure that all code cells follow the [PEP8 standard](https://www.python.org/dev/peps/pep-0008/) for code. The `jupyterlab_code_formatter` tool can be used to format code cells to a consistent style: select each code cell, then click `Edit` and then one of the `Apply X Formatter` options (`YAPF` or `Black` are recommended).
- [x] Include relevant tags in the final notebook cell (refer to the [DEA Tags Index](https://docs.dea.ga.gov.au/genindex.html), and re-use tags if possible)
- [x] Clear all outputs, run notebook from start to finish, and save the notebook in the state where all cells have been sequentially evaluated
- [ ] Test notebook on both the `NCI` and `DEA Sandbox` (flag if not working as part of PR and ask for help to solve if needed)
- [ ] If applicable, update the `Notebook currently compatible with the NCI|DEA Sandbox environment only` line below the notebook title to reflect the environments the notebook is compatible with

### CODE REVIEW

File: Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb [hunk 1]
```diff
@@ -48,11 +48,20 @@
    "outputs": [],
```
Comment (BexDunn) [inline Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb:48]: <p>Consider replacing "early winter" with "late autumn/early winter (MAM/JJA)" - adds the seasonal acronym, for context for those whom winter is hard to place :D </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/724/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='16'/>

Comment (BexDunn) [inline Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb:48]: <p>add a comment about the black line (deseasonalised monthly mean) vs the grey lines (seasonal monthly means)? potentially consider a legend on the RH plot, though dunno where it'd fit...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/724/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='20'/>

Comment (BexDunn) [inline Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb:48]: <p>pretty. I think this goes in the tshirt planning category..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/724/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='25'/>

Comment (BexDunn) [inline Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb:48]: <p>I like this explanation, well done</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/724/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='31'/>

Comment (BexDunn) [inline Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb:48]: <p>also, nice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/724/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='35'/>

Comment (BexDunn) [inline Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb:48]: <p>good - feel free to strengthen this comment if you like eg. by restating that Kati Thanda is wetter during La Niña ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/724/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='39'/>

Comment (MatthewJA) [reply Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb:48]: <p>Thank you!</p>

**Review by BexDunn — APPROVED** [review]
Looks great, I like the changes you've made to the graph explanations.

**Comment by MatthewJA** [comment]
Thank you!

**Pull request merged**